File: src/main/java/de/silencio/activecraftcore/messages/Durations.java
package de.silencio.activecraftcore.messages;

public class Durations extends ActiveCraftCoreMessage {

    public static String MINUTES_15() {
        return getMessage(MessageType.DURATION, "15m");
    }

    public static String HOUR_1() {
        return getMessage(MessageType.DURATION, "1h");
    }

    public static String HOURS_8() {
        return getMessage(MessageType.DURATION, "8h");
    }

    public static String DAY_1() {
        return getMessage(MessageType.DURATION, "1d");
    }

    public static String DAYS_7() {
        return getMessage(MessageType.DURATION, "7d");
    }

    public static String MONTH_1() {
        return getMessage(MessageType.DURATION, "1M");
    }

    public static String PERMANENT() {
        return getMessage(MessageType.DURATION, "permanent");
    }
}

File: src/main/java/org/activecraft/activecraftcore/messages/messagecollections/Durations.kt
package org.activecraft.activecraftcore.messages.messagecollections

import org.activecraft.activecraftcore.messages.MessageSupplier

class Durations(val messageSupplier: MessageSupplier) {
    val minutes15: String get() = messageSupplier.getRawMessage("duration.15m")
    val hour1: String get() = messageSupplier.getRawMessage("duration.1h")
    val hours8: String get() = messageSupplier.getRawMessage("duration.8h")
    val day1: String get() = messageSupplier.getRawMessage("duration.1d")
    val days7: String get() = messageSupplier.getRawMessage("duration.7d")
    val month1: String get() = messageSupplier.getRawMessage("duration.1M")
    val permanent: String get() = messageSupplier.getRawMessage("duration.permanent")
}